whois-query/src/test/java/net/ripe/db/whois/query/pipeline/WhoisEncoderTest.java

```diff
@@ -7,6 +7,7 @@
 import net.ripe.db.whois.common.Message;
 import net.ripe.db.whois.common.domain.ResponseObject;
 import net.ripe.db.whois.query.QueryMessages;
+import org.junit.Ignore;
```

Comment:
Unused import

Reply:
Gone. (My optimize imports setting in IntelliJ had disappeared for some reason)